README.rst

```diff
@@ -235,6 +236,44 @@ Delete
     print(response)
     # <class pybutton.Response >
 
+Customers
+~~~~~~
+
+Create
+''''''
+
+.. code:: python
+
+    import hashlib
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    hashed_email = hashlib.sha256('user@example.com'.lower().strip()).hexdigest()
+
+    response = client.customers.create({
```

Comment:
Does this API support `email_sha256` and `advertising_id`?  I don't see any support for it in our backend ([ref](https://github.com/button/dlcfe/blob/master/lib/customer-manager.js#L96)), not do I see it actually persisted/returned by sorting hat.

cc @mightyguava